src/components/UserProfile/BasicInformationTab/BasicInformationTab.jsx

```diff
@@ -566,7 +566,7 @@ const BasicInformationTab = (props) => {
           <Label>
             {
             userProfile.endDate
-            ? 'End Date ' + moment(userProfile.endDate).format('YYYY-MM-DD')
+            ? 'End Date ' + userProfile.endDate.toLocaleString().split('T')[0]
```

Comment:
It seems that the format isn't consistent with line 557 where it writes
`moment(userProfile.reactivationDate).format('YYYY-MM-DD')`, whereas the original code is consistent. Would this be a potential problem in the future? Or is it better to also change line 557 to `userProfile.reactivationDate.toLocaleString().split('T')[0]`?

Reply:
Thanks, good catch, but I want to remain it here, due to fixing the bug of time zone issue. As for the reactivationDate, we can add another task on aligning it, not for this PR, which is beyond the scope.